src/Facebook/InstantArticles/Elements/InstantArticle.php

```diff
@@ -396,6 +396,7 @@ public function addMetaProperty($property_name, $property_content)
 
     public function render($doctype = '<!doctype html>', $format = false)
     {
+        $doctype = is_null($doctype) ? '<!doctype html>' : $doctype;
```

Comment:
This can be a matter of personal preference, but how about changing the function signature to 
```php:
public function render($doctype = null, $format = false)
```
so you can avoid the recurring '<!doctype html>'.